script.js

```diff
@@ -4,33 +4,211 @@ const logo = document.querySelector('.logo');
 const navMenu = document.querySelector('.nav-menu');
 const hamburger = document.querySelector('.hamburger');
 const navItem = document.querySelectorAll('.nav-item');
+const modal = document.querySelector('.modal');
+const cards = document.querySelector('.card-works');
+const wrapper = document.querySelector('.wrapper');
+const overFlow = document.querySelector('*');
 
-const openModal = () => {
+const openMenu = () => {
   logo.classList.toggle('active');
   navMenu.classList.remove('desktop');
   navMenu.classList.toggle('active');
   hamburger.classList.toggle('active');
 };
 
-const closeModal = () => {
+const closeMenu = () => {
   navMenu.classList.add('desktop');
   logo.classList.remove('active');
   navMenu.classList.remove('active');
   hamburger.classList.remove('active');
 };
 
 hamburger.addEventListener('click', () => {
-  openModal();
+  openMenu();
 });
 
 navItem.forEach((n) => {
   n.addEventListener('click', () => {
-    closeModal();
+    closeMenu();
   });
 });
 
 document.addEventListener('keydown', (e) => {
   if (e.key === 'Escape') {
-    closeModal();
+    closeMenu();
+  }
+});
+
+const projects = [
+  {
+    title: 'Multi-Post Stories',
+    technologies: ['html', 'css', 'ruby on rails'],
+    imageURL: './assets/images/content/Img-Placeholder.svg',
+    content:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took astandard dummy text.",
+    seeLiveLink: 'https://binhussen.github.io/portfolio/',
+    seeSourceLink: 'https://github.com/binhussen/portfolio',
```

Comment:
- [ ] kindly follow the figma version and bring down the seelive/seesource buttons
![Screenshot (269)_LI](https://user-images.githubusercontent.com/65298048/180247969-ec941a17-f319-4b3b-8dc8-c7ccf6df1414.jpg)

Reply:
Thank you @tufoinnkuo10 for your feedback, I will modify it.